tests/check_new_syntax.py

```diff
@@ -46,11 +46,13 @@ def visit_AsyncFunctionDef(self, node: ast.AsyncFunctionDef) -> None:
 
     class ObjectClassdefFinder(ast.NodeVisitor):
         def visit_ClassDef(self, node: ast.ClassDef) -> None:
-            if any(isinstance(base, ast.Name) and base.id == "object" for base in node.bases):
-                errors.append(
-                    f"{path}:{node.lineno}: Do not inherit from `object` explicitly, "
-                    f"as all classes implicitly inherit from `object` in Python 3"
-                )
+            # Temporarily disable this check for two ex-Python 2 stubs.
+            if "paramiko" not in str(path) and "cryptography" not in str(path):
```

Comment:
I'll send a separate PR once this is merged to fix those problems and re-enable this check.